etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -12,6 +12,7 @@
 import numpy as np
 
 from etna import SETTINGS
+from etna.models.utils import get_known_hash, verify_file_hash
```

Comment:
It's seems that transforms should not depend on models module in such way. It's unnecessary coupling of modules that not core but domain specific